src/lib.rs

```diff
@@ -409,12 +465,8 @@ fn substitute(input: &[u8]) -> (Vec<u8>, Option<Handler>) {
         // Report the `main` handler for the possible information disclosure risk, then assume
         // empty/default string expansion in case this was just obfuscatory.
         (default.into(), Some(Handler::Main))
-    } else if let Some(_) = strip_lower_ascii_prefix(value, b"date:") {
-        // `${date:...}` expands a format string using the current time. There isn't a way to
-        // select substrings, so month/day strings aren't useful for building an unaccepatble
-        // handler. Assume that if `${date:...}` is showing up in an interesting place, it will be
-        // with some format string that expands to empty string.
```

Comment:
i think "isn't a way to select substrings" is still an important thing to note somewhere around date handling, because that's the entire way we can justify replacing formatting characters with `" "`, rather than trying to do _something_ about month/day names.